mongodb/plan.sh

```diff
@@ -0,0 +1,57 @@
+pkg_name=mongodb
+pkg_origin=core
+pkg_version=3.2.9
+pkg_maintainer="Some Maintainer <somemaintainer@example.com>"
```

Comment:
Habitat packages(core) have pkg_maintainer set to `The Habitat Maintainers <humans@habitat.sh>`